sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -349,7 +349,8 @@ def test_external_transform_finder_non_leaf(self):
     self.assertTrue(pipeline.contains_external_transforms)
 
   def test_external_transform_finder_leaf(self):
-    pipeline = beam.Pipeline()
+    opts = PipelineOptions(['--runner=DirectRunner'])
```

Comment:
What problem is this causing? This should be the default, so I'm surprised this helps